### PR TITLE
Added the right margin and padding to the 'maatregelen' page.

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
On the 'Maatregelen' page there are some visual differences on the top two headings which do not comply with the overall look. There needed to be a coherent style. So the styling was adjusted.

### CODE REVIEW

**Review by VWSCoronaDashboard19 — APPROVED** [review]
LGTM 👍

File: packages/app/src/components/page-information-block/page-information-block.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export function PageInformationBlock({
   ) : null;
 
   return (
-    <Box as="header" id={id} spacing={{ _: 3, md: 4 }}>
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/components/page-information-block/page-information-block.tsx:70]: This component gets used on a lot of pages, removing this spacing everywhere might not be a good idea.

File: packages/app/src/components/tile-list.tsx [hunk 2]
```diff
@@ -2,12 +2,24 @@ import css from '@styled-system/css';
...
+  ({ hasActiveWarningTile}) => {
+    return css({
+      pt: hasActiveWarningTile ? 4 : asResponsiveArray({ _: 3, md: 5 }),
+      px: asResponsiveArray({ _: 3, sm: 5 }),
+    } )
+  },
+  ({needsMargin}) => {
+    return needsMargin ? spacing : undefined;
+  },
+);
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/components/tile-list.tsx:25]: Removing the spacing has quite an impact also on all the metric pages, this is used for giving every component some bottom margin.

File: packages/app/src/components/tile.tsx [hunk 3]
```diff
@@ -26,15 +29,17 @@ const StyledTile = styled.article<{
...
   noPadding: boolean;
+  isFirstElement: boolean;
 }>((x) =>
   css({
     position: 'relative',
     display: 'flex',
     flexDirection: 'column',
-    pt: x.noPadding ? undefined : 4,
-    pb: x.noPadding ? undefined : asResponsiveArray({ _: 3, sm: 4 }),
+    pt: x.noPadding ? undefined : x.isFirstElement ? 4 : '.5rem',
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/components/tile.tsx:38]: The `.5rem` is also in the theme styling

File: packages/app/src/components/tile.tsx [hunk 4]
```diff
@@ -26,15 +29,17 @@ const StyledTile = styled.article<{
...
+    pb: x.noPadding ? undefined : x.isFirstElement ? asResponsiveArray({ _: 3, sm: 4 }) : '2rem',
     height: x.height,
     backgroundColor: 'white',
     borderTop: x.noBorder ? undefined : 'solid 2px lightGray',
-  })
+    mb: x.isFirstElement ? undefined : '1rem',
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/components/tile.tsx:43]: Same goes for the `1rem` here

**Comment by VWSCoronaDashboard14** [comment]
There is also a maatregelen page on safety region level

File: packages/app/src/pages/landelijk/maatregelen.tsx [hunk 5]
```diff
@@ -71,10 +71,10 @@ const NationalRestrictions = (props: StaticProps<typeof getStaticProps>) => {
   return (
     <Layout {...metadata} lastGenerated={lastGenerated}>
       <NlLayout>
-        <TileList>
+        <TileList needsMargin={false}>
           <PageInformationBlock title={siteText.nationaal_maatregelen.titel} />
-          <Tile>
-            <Box spacing={3}>
+          <Tile noBorder={true} isFirstElement={false}>
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/pages/landelijk/maatregelen.tsx:76]: Introducing a new prop that only gets used on 1 page, we're might be better off by just using a `<Box as="article">` with some extra CSS for the measurements page.

**Review by VWSCoronaDashboard1 — DISMISSED** [review]
I think @VWSCoronaDashboard14 has already pointed out the main flaws with these changes, so I don't think I have much to add.

File: packages/app/src/pages/landelijk/maatregelen.tsx [hunk 6]
```diff
@@ -67,25 +65,24 @@ const NationalRestrictions = (props: StaticProps<typeof getStaticProps>) => {
...
-          <Tile>
-            <Box spacing={3}>
-              <Heading level={3}>{lockdown.title}</Heading>
-              <LockdownTable
-                data={lockdown}
-                level={data.risk_level.last_value.risk_level}
-              />
-            </Box>
-          </Tile>
+          <Box as="header" spacing={1}>
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/pages/landelijk/maatregelen.tsx:68]: A `spacing={4}` would do justice here

File: packages/app/src/pages/landelijk/maatregelen.tsx [hunk 7]
```diff
@@ -67,25 +65,24 @@ const NationalRestrictions = (props: StaticProps<typeof getStaticProps>) => {
...
+            <Heading
+              level={1}
+              as="h2"
+            >
+              {siteText.nationaal_maatregelen.titel}
+            </Heading>
+            {lockdown.message.description ? (
+              <RichContent blocks={lockdown.message.description} />
+            ) : null}
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/pages/landelijk/maatregelen.tsx:77]: We try to cap the width of the paragraph text to the maxWidthText from the theming, reads a bit easier for the users.

```
 {lockdown.message.description ? (
    <Box maxWidth='maxWidthText'>
        <RichContent blocks={lockdown.message.description} />
   </Box>
) : null} 
```

File: packages/app/src/pages/veiligheidsregio/[code]/maatregelen.tsx [hunk 8]
```diff
@@ -90,39 +87,30 @@ const RegionalRestrictions = (props: StaticProps<typeof getStaticProps>) => {
...
+                siteText.veiligheidsregio_maatregelen.titel,
+                {
+                  safetyRegionName: vrName,
+                })
+              }
+            </Heading>
+            {lockdown.message.description ? (
+              <RichContent blocks={lockdown.message.description} />
+            ) : null}
+          </Box>
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/pages/veiligheidsregio/[code]/maatregelen.tsx:105]: Same goes for here

**Review by VWSCoronaDashboard14 — APPROVED** [review]

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Pull request merged**